src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx

```diff
@@ -1,17 +1,23 @@
+import { Fraction } from '@kyberswap/ks-sdk-core'
 import { Trans, t } from '@lingui/macro'
 import dayjs from 'dayjs'
+import JSBI from 'jsbi'
 import { useMemo } from 'react'
 import { Text } from 'rebass'
 import styled, { css } from 'styled-components'
 
 import Divider from 'components/Divider'
 import InfoHelper from 'components/InfoHelper'
-import { RowBetween } from 'components/Row'
+import Row, { RowBetween } from 'components/Row'
 import { useActiveWeb3React } from 'hooks'
 import { useVotingInfo } from 'hooks/kyberdao'
 import { ProposalDetail } from 'hooks/kyberdao/types'
 import useTheme from 'hooks/useTheme'
 
+import WarningIcon from '../../../../components/Icons/WarningIcon'
+import { MouseoverTooltip } from '../../../../components/Tooltip'
+import { BIPS_BASE } from '../../../../constants'
```

Comment:
please use absolute import

Reply:
Fixed. Can't use absolute for constants.

Reply:
Fixed.